core/gesture.js

```diff
@@ -41,8 +41,9 @@ goog.require('goog.math.Coordinate');
 /**
  * NB: In this file "start" refers to touchstart, mousedown, and pointerstart
```

Comment:
@rachel-fenichel : What is "NB" suppose to mean, and should this even be a doc comment (`/** ... */`)?